example/settings.py

```diff
@@ -82,9 +82,15 @@
 
 ES_DISABLED = False
 
-ES_URLS = ['http://localhost:9200']
+ES_URLS = ['http://elasticsearch.local:9200']
```

Comment:
We can forward dc elasticsearch to localhost:9200 and I believe should since that is how elasticsearch is run anyway, and like @mparent61 said, this isn't a docker project...